### PR TITLE
[wespeaker]merge onnx cpu and gpu code, add infer code

### PR DESCRIPTION
**Pull request by wsstriving**
When I test the export and infer code, I need to set `opset_version=13` instead of `opset_version=14` as in this commit. The reason is that I failed to run the infer code using the latest onnxruntime and use version 1.6.0 as a workaround.

### CODE REVIEW

File: wespeaker/bin/infer_onnx.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    feats = compute_fbank(wav_path).numpy()
+
+    embedding = session.run(
+        output_names=['embs'],
+        input_feed={
+            'feats': feats[None, :]
+        }
+    )[1]
+    print(embedding.shape)
+
```
Comment (Slyne) [inline wespeaker/bin/infer_onnx.py:68]:
````
def main():
    args = get_args()
    session = ort.InferenceSession(args.onnx_path)
    wav_path = args.wav_path
    feats = compute_fbank(wav_path)
    # add batch dimension
    feats = feats.unsqueeze(0).numpy()

    embeddings = session.run(
        output_names=['embs'],
        input_feed={
            'feats': feats
        }
    )
    print(embeddings[0].shape)
```

Test opset14 and onnxruntime-1.11 (latest staple version)

**Review by Slyne — COMMENTED** [review]
1. Is the onnx model supposed to have two outputs (emb_a and emb_b) or just (emb_b) ?
2. I've tested the current 'export_onnx.py' which uses opset14 and the exported onnx model works fine in onnxruntime-1.11. But I change the input feature to be [B, T, F].

File: wespeaker/bin/export_onnx.py [hunk 2]
```diff
@@ -28,6 +29,7 @@ def get_args():
     parser = argparse.ArgumentParser(description='export your script model')
     parser.add_argument('--config', required=True, help='config file')
     parser.add_argument('--checkpoint', required=True, help='checkpoint model')
+    parser.add_argument('--type', required=True, help='cpu|cuda')
```
Comment (Slyne) [inline wespeaker/bin/export_onnx.py:32]: --type  can be removed. no need to use `cuda` when exporting pytorch model to onnx model.   We may use cuda_provider when inferencing with onnxruntime gpu: https://onnxruntime.ai/docs/execution-providers/#use-execution-providers      I've double-checked the onnx model without `--type=cuda` can be safely exported in tensorrt.

**Review by Slyne — COMMENTED** [review]
Suggestion: remove --type

**Comment by JiJiJiang** [comment]
> 1. Is the onnx model supposed to have two outputs (emb_a and emb_b) or just (emb_b) ?
> 2. I've tested the current 'export_onnx.py' which uses opset14 and the exported onnx model works fine in onnxruntime-1.11. But I change the input feature to be [B, T, F].



> 

1. Only emb_b should be fine. We are considering removing emb_a in the output.

File: wespeaker/bin/export_onnx.py (hunk 2 above)
Comment (wsstriving) [reply wespeaker/bin/export_onnx.py:32]: ok, I will remove it

**Review by JiJiJiang — APPROVED** [review]

**Pull request merged**